website/src/MDPage.js

```diff
@@ -29,9 +27,20 @@ const getHeadingForLevel = (level) => {
   }
 };
 
+const isDefined = (obj) => obj !== undefined;
+
+const getDocsMD = (docPath, navigation) => {
+  const { state } = navigation;
+  const docVersion = (isDefined(state) && isDefined(state.params))
```

Comment:
AFAIK `navigation.state` is always defined. We can only check for `state.params && state.params.version`